tests/test_routers.py

```diff
@@ -75,9 +77,25 @@ def regex_url_path_detail(self, request, *args, **kwargs):
         return Response({'pk': pk, 'kwarg': kwarg})
 
 
+class UrlPathViewSet(viewsets.ViewSet):
+    @action(detail=False, url_path='list/<int:kwarg>')
+    def url_path_list(self, request, *args, **kwargs):
+        kwarg = self.kwargs.get('kwarg', '')
+        return Response({'kwarg': kwarg})
+
+    @action(detail=True, url_path='detail/<int:kwarg>')
+    def url_path_detail(self, request, *args, **kwargs):
+        pk = self.kwargs.get('pk', '')
+        kwarg = self.kwargs.get('kwarg', '')
+        return Response({'pk': pk, 'kwarg': kwarg})
+
+
 notes_router = SimpleRouter()
 notes_router.register(r'notes', NoteViewSet)
 
+notes_path_router = SimpleRouter(use_regex_path=False)
```

Comment:
can we add another test case which use DeafultRouter please?

Reply:
It should not be a problem to replicate that test also with the other router.